core-redirects-cae/src/main/java/com/tallence/core/redirects/cae/filter/RedirectFilter.java

```diff
@@ -235,7 +237,28 @@ private void sendPermanentRedirect(HttpServletRequest request, HttpServletRespon
     response.setHeader(HttpHeaders.PRAGMA, "no-cache");
     response.setDateHeader(HttpHeaders.EXPIRES, 0);
     ContentBean targetBean = contentBeanFactory.createBeanFor(target.getTarget());
-    response.setHeader(HttpHeaders.LOCATION, linkFormatter.formatLink(targetBean, null, request, response, true));
+
+    String targetLink = linkFormatter.formatLink(targetBean, null, request, response, true);
+
+    try {
+      targetLink = handleSourceParams(request, targetLink);
+    } catch (Exception e) {
```

Comment:
Catch auf Exception?

Reply:
Da die Url hier explizit nicht neu enkodiert wird, kann es sein, dass der UrlBuilder bei Sonderzeichen in speziell konfigurierten Projekt-Setups eine IllegalArgumentException schmeißt.
Die ist m.E. zu unspezifisch um sie explizit zu fangen, da sie ja auch tief im SpringUrlComponentsBuilder geworfen wird.

Reply:
Wie besprochen, wird eine RuntimeException